src/backend/distributed/sql/udfs/get_rebalance_table_shards_plan/latest.sql

```diff
@@ -10,7 +10,8 @@ CREATE OR REPLACE FUNCTION pg_catalog.get_rebalance_table_shards_plan(
         max_shard_moves int default 1000000,
         excluded_shard_list bigint[] default '{}',
         drain_only boolean default false,
-        rebalance_strategy name default NULL
+        rebalance_strategy name default NULL,
+        utilization_improvement_threshold float4 DEFAULT 0.5
```

Comment:
as discussed. This should definitely be stored on the strategy for persistence. I am unsure if we want to keep it as an override possibility here.

It could be beneficial during experimentation in finding the right cutoff (50% feels like a good default but not much investigation has gone in here). However, not storing this could cause other administrators (or future background jobs) to undo a newly found cutoff threshold unintentionally.

Now that I think of it, we might want to keep it here as a user parameter and remove the parameter from the actual rebalance function only. This allows an experienced administrator to experiment with the plan that gets created, but requires the storing on the strategy to use from the rebalancer.